src/main/java/frc/robot/utils/Utils.java

```diff
@@ -26,4 +26,23 @@ public static double rotationalDeadband(double input, double threshold) {
         return (input - (Math.signum(input) * threshold)) / (1 - threshold);
     }
 
+    /**
+     * Converts minutes into seconds.
+     * @param minutes is the input of minutes. [min]

```

Comment:
Adam taught us documentation a little bit differently. @adamv17 could you take a look?

Reply:
OK, that's fine if you have your own convention. But just FYI, the [Javadoc article on Oracle's website](https://www.oracle.com/il-en/technical-resources/articles/java/javadoc-tool.html#format) states otherwise.